cpp/node_module/node_module.cpp

```diff
@@ -5,6 +5,10 @@
 extern "C" int mgp_init_module(struct mgp_module *module, struct mgp_memory *memory) {
   try {
     mgp::memory = memory;
+    AddProcedure(Node::RelExists, Node::kProcedureRelExists, mgp::ProcedureType::Read,
+                 {mgp::Parameter(Node::kArgumentsNode, mgp::Type::Node),
+                  mgp::Parameter(Node::kArgumentsPattern, mgp::Type::String, "")},
```

Comment:
We should also accommodate the case where multiple patterns are given (#300 and #301 use a list for the second parameter).

Reply:
That is the only difference between node.relationship_exists and node.relationships_exist, if I do this, those will be the same procedures right? https://github.com/memgraph/mage/pull/298/files

Reply:
The return would still be different. `relationships_exist` returns the existence of a relationship for _each_ of the given patterns, whereas this method returns whether there is a relationship that fits `at least one` of the given pattern(s).